test/testunusedfunctions.cpp

```diff
@@ -180,7 +181,7 @@ class TestUnusedFunctions : public TestFixture {
               "    if (cond) ;\n"
```

Comment:
Maybe also add the missing `cond` variable when fixing this up.

Reply:
Ah - too late.

Reply:
`cond` could be an unknown global variable.
The `return2()` test is not compilable though.